device/ruijie/x86_64-ruijie_b6510-48vs8cq-r0/minigraph.xml

```diff
@@ -0,0 +1,1199 @@
+<DeviceMiniGraph xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="Microsoft.Search.Autopilot.Evolution">
```

Comment:
this is no longer needed, please remove.

Reply:
ok,done!